### PR TITLE
Charmhub requirements

### PR DESCRIPTION
**Pull request by Gleland**
add charmcraft and metadata changes to build with charmcraft pack successfully

### CODE REVIEW

File: charmcraft.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+          channel: "20.04"
+          architectures:
+              - amd64
+        - name: ubuntu
+          channel: "18.04"
+          architectures:
+              - amd64
+parts:
+  charm:
+    charm-python-packages: [setuptools < 58]
```
Comment (afreiberger) [inline charmcraft.yaml]: why is this necessary? I don't see jinja2 or charmhelpers in requirements.txt

**Review by afreiberger — CHANGES_REQUESTED** [review]

File: charmcraft.yaml (hunk 1 above)
Comment (Gleland) [reply charmcraft.yaml]: good catch, thanks! i've removed this, and verified that it doesn't cause an issue when I build locally.

File: metadata.yaml [hunk 2]
```diff
@@ -6,7 +6,7 @@ description: |
   A client provides a way to sync or act on changes in a remote stream.
 summary: |
   Simple Streams repository
-series: [bionic]
+series: [bionic,focal]
```
Comment (afreiberger) [inline metadata.yaml]: Would you mind terribly using a list indent or add a space?

File: charmcraft.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+type: charm
+bases:
+    - build-on:
+        - name: ubuntu
+          channel: "20.04"
+          architectures: ["amd64"]
```
Comment (afreiberger) [inline charmcraft.yaml]: Not sure if I've approved this before, but we should be consistent in the same yaml file whether we use list indents or list notation.

File: metadata.yaml (hunk 2 above)
Comment (Gleland) [reply metadata.yaml]: agreed! thanks for catching this.

File: charmcraft.yaml (hunk 3 above)
Comment (Gleland) [reply charmcraft.yaml]: agreed! thanks, this has been updated.

**Review by afreiberger — APPROVED** [review]
LGTM, thanks for taking the extra time to clear that up!
